src/firefly/js/charts/ui/PlotlyChartArea.jsx

```diff
@@ -25,6 +26,12 @@ function adjustLayout(layout={}) {
     set(layout, 'yaxis.ticksuffix', hasOppositeY ? '  ' : '');
     set(layout, 'margin.b', hasOppositeX ? MIN_MARGIN_PX: X_TICKLBL_PX);
     set(layout, 'margin.t', hasOppositeX ? X_TICKLBL_PX: MIN_MARGIN_PX + (hasTitle ? TITLE_PX: 0));
+
+    // make background same as app's background color
+    const bgSurface = getTheme()?.palette?.background?.surface?.split(',')[1].slice(0,-1);     // plotly will only take a color string
+    if (!layout?.paper_bgcolor) set(layout, 'paper_bgcolor', bgSurface);
+    if (!layout?.plot_bgcolor)  set(layout, 'plot_bgcolor', bgSurface);
```

Comment:
You could eventually start checking for dark mode and change it. I did that with hips.

Reply:
`background.surface` supports both mode.  but, we need to reload browser.  i can add useColorMode so that it watches for mode change in my next PR.